sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/DocumentModelAdministrationLiveTests.cs

```diff
@@ -38,7 +38,7 @@ public DocumentModelAdministrationLiveTests(bool isAsync, DocumentAnalysisClient
         #region Build
 
         [RecordedTest]
-        public async Task BuildModelCanAuthenticateWithTokenCredential()
+        public async Task BuildDocumentModelCanAuthenticateWithTokenCredential()
```

Comment:
Renaming all "build" tests from `BuildModel` to `BuildDocumentModel` to align with the service terminology. I still plan to revisit the "build" tests in another PR, though.